app/src/main/res/values/environment_config.xml

```diff
@@ -25,5 +25,6 @@
 
     <string name="orca_api_base_url">https://api.orca.so/</string>
 
-    <string name="notification_service_url">https://push-service.wallet.p2p.org/</string>
+    <string name="notificationServiceBaseUrl">https://push-service.wallet.p2p.org/</string>
+    <string name="notificationServiceTestBaseUrl">http://35.234.120.240:9090/</string>
```

Comment:
is it working with http? 
Are you sure we don't need to add network config?

Reply:
network config ?

Reply:
it should works as far as QA checked 

also it was done same way lire feerelayer url changes 

but we can do it in some other way 

just let me know

Reply:
yes, if I remember correctly we should have issues with `http` requests, not `https`. 

For such cases, we had to use `useClearTextTraffic` or smth like that, or add network_config.xml and include it inside the manifest

Reply:
hmmm... ok I will add this address to config but it is strange that it was ok on testing :(

Reply:
@1sakov  done